backend/config/routes.rb

```diff
@@ -168,9 +168,12 @@
       member do
         get :orders
         get :items
-        get :addresses
         put :addresses
       end
+
+      get 'addresses', action: :show, controller: 'addresses', as: :addresses
+      put 'addresses/update', action: :update, controller: 'addresses', as: :addresses_update
```

Comment:
As mentioned [here](https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing), it's probably fine to skip the `_update` part in the `as:` argument. The PUT verb is already telling routes which action to take. Can we try without it please?